modules/setup/application/views/scripts/form/setup-requirements.phtml

```diff
@@ -13,6 +13,10 @@ if (! $form->getWizard()->getRequirements()->fulfilled()) {
 <h1><?= ucwords($moduleName) . ' ' . $this->translate('Module'); ?></h1>
 <?= $wizard->getRequirements(); ?>
 <?php endforeach ?>
+<?php foreach ($form->getWizard()->getPage('setup_modules')->getDependecyRequirements() as $moduleName => $wizard): ?>
+    <h1><?= ucwords($moduleName) . ' ' . $this->translate('Module Dependencies'); ?></h1>
+    <?= $wizard->getRequirements(); ?>
+<?php endforeach ?>
```

Comment:
I wonder why that's necessary, can't you just make `getModuleWizards()` from above to automatically return the dependency wizards?